### DISCUSSION TITLE
Shared Environment Variables VSCODE_APPDATA and VSCODE_EXTENSIONS

### DISCUSSION THREAD

**Discussion by juanpickselov** [microsoft/azuredatastudio#16006] [Q&A]
If I set the environment variables VSCODE_APPDATA and VSCODE_EXTENSIONS so I don't have to use the --user-data-dir and --extensions-dir on command line in Visual Studio Code, Azure Data Studio picks them up because they're in its source code. Is this something you're going to keep the same, or will different named variables be created for Azure Data Studio?

**Comment by Charles-Gagnon** [comment]
We aren't planning on changing this currently- doing so brings in additional conflicts we have to resolve when merging in their code and adds risk if we miss updating one somewhere so generally isn't worth the effort/risk.

**Reply by juanpickselov** [reply]
sharing them across different apps doesn't help either
